### PR TITLE
Task/configurable protocol

### PR DESCRIPTION
**Pull request by dmoranj**
ADD Configurable environment variables for HTTP protocol.

### CODE REVIEW

**Comment by jcalderin** [comment]
LGTM

File: bin/pepProxy [hunk 1]
```diff
@@ -36,8 +36,10 @@ function loadConfiguration() {
         'LOG_LEVEL',
         'ACCESS_HOST',
         'ACCESS_PORT',
+        'ACCESS_PROTOCOL',
```
Comment (fgalan) [inline bin/pepProxy]: I think (but not sure) this is a table in the doc with a list of env variables. Have that list being aligned with the new added variables in the code?

Comment (dmoranj) [inline bin/pepProxy]: Fixe din f921565

**Comment by fgalan** [comment]
LGTM

**Pull request merged**